### DISCUSSION TITLE
Install instructions for Ubuntu failed on lunar version

### DISCUSSION THREAD

**Discussion by L1st3r** [aquasecurity/trivy#5799] [Documentation]
### Description

Followed the instructions for adding the repo to apt. When I did an apt update, I got this returned:

E: The repository 'https://aquasecurity.github.io/trivy-repo/deb lunar Release' does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

### Suggestions

Wondering if you can add that to the repo for us 23.04 users! Otherwise if I have to install via .deb, I'm never going to remember to update it. Thank you!

**Comment by DmitriyLewen** [comment]
Hello @L1st3r 
Thanks for your report!

Created #5810 for this task.

Regards, Dmitriy
